src/mixins/filing-template-mixin.ts

```diff
@@ -400,7 +401,7 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     this.setRegistrationBusinessAddress(draftFiling.registration.offices?.businessOffice)
 
     // restore Business Type
-    this.setRegistrationBusinessType(draftFiling.registration.businessType || BusinessTypes.SP)
+    this.setRegistrationBusinessType(draftFiling.registration.businessType)
```

Comment:
I thought there was a `|| null` here before? But it looks like this will also work (ie, if businessType is not in the filing object then undefined will be assigned).

For consistency, would leaving off the fallback work for Business Type Confirm below?

Reply:
We will use false for Business Type Confirm below, so this will be okay.

Reply:
1. Did you remove `|| null` from line 404, and if so, why?

2. What about the case where `businessTypeConfirm` is not a field in the draft filing (eg, a brand new filing)? That's where the fallback makes sense.

Reply:
I unresolved this because my 2 points above have not been addressed.

Reply:
I changed it to || false in my latest commit: c77ff41.